diff --git a/src/AppBundle/DataFixtures/ORM/AppFixtures.php b/src/AppBundle/DataFixtures/ORM/AppFixtures.php
index 49b873f..37e2aec 100644
--- a/src/AppBundle/DataFixtures/ORM/AppFixtures.php
+++ b/src/AppBundle/DataFixtures/ORM/AppFixtures.php
@@ -16,4 +16,20 @@ class AppFixtures extends DataFixtureLoader
             __DIR__ . '/characters.yml',
         );
     }
+
+    public function characterName()
+    {
+        $names = array(
+            'Mario',
+            'Luigi',
+            'Sonic',
+            'Pikachu',
+            'Link',
+            'Lara Croft',
+            'Trogdor',
+            'Pac-Man',
+        );
+
+        return $names[array_rand($names)];
+    }
 }
diff --git a/src/AppBundle/DataFixtures/ORM/characters.yml b/src/AppBundle/DataFixtures/ORM/characters.yml
index 64d7af1..288b484 100644
--- a/src/AppBundle/DataFixtures/ORM/characters.yml
+++ b/src/AppBundle/DataFixtures/ORM/characters.yml
@@ -7,7 +7,7 @@ AppBundle\Entity\Character:
         tagLine: Yoshi!
 
     character{2..10}:
-        name: Super <firstName()>
+        name: <characterName()>
         realName: <name()>
         highScore: <numberBetween(100, 99999)>
         email: <email()>
