diff --git a/src/AppBundle/DataFixtures/ORM/AppFixtures.php b/src/AppBundle/DataFixtures/ORM/AppFixtures.php
index e3f2787..c7fcf77 100644
--- a/src/AppBundle/DataFixtures/ORM/AppFixtures.php
+++ b/src/AppBundle/DataFixtures/ORM/AppFixtures.php
@@ -33,4 +33,16 @@ class AppFixtures extends DataFixtureLoader
 
         return $names[array_rand($names)];
     }
+
+    public function avatar()
+    {
+        $filenames = array(
+            'kitten1.jpg',
+            'kitten2.jpg',
+            'kitten3.jpg',
+            'kitten4.jpg',
+        );
+
+        return $filenames[array_rand($filenames)];
+    }
 }
diff --git a/src/AppBundle/DataFixtures/ORM/characters.yml b/src/AppBundle/DataFixtures/ORM/characters.yml
index 1e94042..b55a023 100644
--- a/src/AppBundle/DataFixtures/ORM/characters.yml
+++ b/src/AppBundle/DataFixtures/ORM/characters.yml
@@ -14,4 +14,4 @@ AppBundle\Entity\Character:
         email: <email()>
         tagLine: 80%? <sentence()>
         universe: @universe_*
-        avatarFilename:
+        avatarFilename: <avatar()>
