diff --git a/src/AppBundle/DataFixtures/ORM/AppFixtures.php b/src/AppBundle/DataFixtures/ORM/AppFixtures.php
index 178a322..a673c57 100644
--- a/src/AppBundle/DataFixtures/ORM/AppFixtures.php
+++ b/src/AppBundle/DataFixtures/ORM/AppFixtures.php
@@ -49,7 +49,7 @@ class AppFixtures extends DataFixtureLoader
     protected function getProcessors()
     {
         return array(
-            new AvatarProcessor()
+            new AvatarProcessor($this->container->get('logger'))
         );
     }
 }
diff --git a/src/AppBundle/DataFixtures/ORM/AvatarProcessor.php b/src/AppBundle/DataFixtures/ORM/AvatarProcessor.php
index 5bc3b59..11cbd7d 100644
--- a/src/AppBundle/DataFixtures/ORM/AvatarProcessor.php
+++ b/src/AppBundle/DataFixtures/ORM/AvatarProcessor.php
@@ -4,10 +4,18 @@ namespace AppBundle\DataFixtures\ORM;
 
 use AppBundle\Entity\Character;
 use Nelmio\Alice\ProcessorInterface;
+use Psr\Log\LoggerInterface;
 use Symfony\Component\Filesystem\Filesystem;
 
 class AvatarProcessor implements ProcessorInterface
 {
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
     /**
      * Processes an object before it is persisted to DB
      *
@@ -33,6 +41,12 @@ class AvatarProcessor implements ProcessorInterface
             true
         );
 
+        $this->logger->debug(sprintf(
+            'Character %s using filename %s from %s',
+            $object->getName(),
+            $targetFilename,
+            $object->getAvatarFilename()
+        ));
         $object->setAvatarFilename($targetFilename);
     }
 
